.github/workflows/label-check.yml

```diff
@@ -8,7 +8,8 @@ jobs:
   label-check:
     runs-on: ubuntu-latest
     steps:
-      - uses: NathanielHill/check-pr-label-action@v4.0.0
+      - uses: jesusvasquez333/verify-pr-label-action@v1.4.0
```

Comment:
It alters the check-pr-label action to use the version that's the upstream of the one we currently use (which is an out of date fork)

Previous: https://github.com/NathanielHill/check-pr-label-action

Suggested: https://github.com/jesusvasquez333/verify-pr-label-action